.buildkite/generate-tags-pipeline.sh

```diff
@@ -14,7 +14,7 @@ ymlappend () {
 }
 
 # we don't bottle meta-formulas that contain only services
```

Comment:
Some command in `.buildkite/pipeline-raw.yaml` (see "in order to ensure package generation sanity") seem to be using still `011` binaries.